KeycloakServer in keycloak-test-suite-utils fails with NoClassDefFoundError: kotlin/jvm/internal/Intrinsics

### Describe the bug

This is due to https://github.com/keycloak/keycloak-ui/pull/3095 adding Kotlin as a dependency, and all dependencies are excluded in `testsuite/utils/pom.xml`.

```
Exception in thread "main" java.lang.RuntimeException: RESTEASY003325: Failed to construct public org.keycloak.services.resources.KeycloakApplication()
....
Caused by: java.lang.RuntimeException: java.lang.NoClassDefFoundError: kotlin/jvm/internal/Intrinsics
    at org.keycloak.testsuite.TestPlatform.exit(TestPlatform.java:45)
    at org.keycloak.services.resources.KeycloakApplication.<init>(KeycloakApplication.java:119)
    at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
    at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
    at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
    at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
    at org.jboss.resteasy.core.ConstructorInjectorImpl.constructOutsideRequest(ConstructorInjectorImpl.java:225)
    ... 21 more
Caused by: java.lang.NoClassDefFoundError: kotlin/jvm/internal/Intrinsics
    at org.keycloak.admin.ui.rest.AdminUIRestEndpointProvider.init(AdminUIRestEndpointProvider.kt)
    at org.keycloak.services.DefaultKeycloakSessionFactory.loadFactories(DefaultKeycloakSessionFactory.java:297)
    at org.keycloak.services.DefaultKeycloakSessionFactory.init(DefaultKeycloakSessionFactory.java:106)
    at org.keycloak.services.resources.KeycloakApplication.createSessionFactory(KeycloakApplication.java:232)
    at org.keycloak.services.resources.KeycloakApplication.startup(KeycloakApplication.java:125)
    at org.keycloak.testsuite.TestPlatform.onStartup(TestPlatform.java:36)
    at org.keycloak.services.resources.KeycloakApplication.<init>(KeycloakApplication.java:115)
    ... 26 more
Caused by: java.lang.ClassNotFoundException: kotlin.jvm.internal.Intrinsics
    at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
    at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
    at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
    ... 33 more
```

### Version

999-SNAPSHOT

### Expected behavior

KeycloakServer starts without problems

### Actual behavior

It fails with the exception shown above

### How to Reproduce?

_No response_

### Anything else?

_No response_

--- a/testsuite/integration-arquillian/pom.xml
+++ b/testsuite/integration-arquillian/pom.xml
@@ -216,12 +216,6 @@
                 <groupId>org.keycloak</groupId>
                 <artifactId>keycloak-admin-ui</artifactId>
                 <version>${keycloak.admin-ui.version}</version>
-                <exclusions>
-                    <exclusion>
-                        <groupId>*</groupId>
-                        <artifactId>*</artifactId>
-                    </exclusion>
-                </exclusions>
             </dependency>
         </dependencies>
     </dependencyManagement>

--- a/testsuite/utils/pom.xml
+++ b/testsuite/utils/pom.xml
@@ -57,12 +57,6 @@
         <dependency>
             <groupId>org.keycloak</groupId>
             <artifactId>keycloak-admin-ui</artifactId>
-            <exclusions>
-                <exclusion>
-                    <groupId>*</groupId>
-                    <artifactId>*</artifactId>
-                </exclusion>
-            </exclusions>
         </dependency>
         <dependency>
             <groupId>org.keycloak</groupId>

